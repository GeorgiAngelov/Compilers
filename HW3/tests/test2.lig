var x : int = y;
var y : int = y;
var z : [int] = [x, y, z];
