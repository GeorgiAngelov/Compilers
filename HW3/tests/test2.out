(decls
   (decl-var:ok (id x) (type-int) (id:int y))
   (decl-var:ok (id y) (type-int) (id:int y))
   (decl-var:NONE (id z) (type-array (type-int)) (array-lit:[NONE] (exps (id:int x) (id:int y) (id:[int] z)))))
Function main() not defined or has wrong type!
Typechecking failed!
